File: quiz3/swift/quizQ8.swift
// 8. Write a program to convert a given number to hours and minutes.

print("Please enter a number you want converted into hours and minutes")
var givenNumber = Int(readLine()!) ?? 0
var hours = givenNumber / 60
var minutes = givenNumber % 60

print("\(givenNumber) is \(hours) Hours and \(minutes) Minutes")

File: quiz3/php/quizQ8.php

<?php
// 8. Write a program to convert a given number to hours and minutes.

$givenNumber = readline("Please enter the number you would like converted into hours and minutes: " );
$hours = floor($givenNumber / 60);
$minutes = $givenNumber % 60;
print("$givenNumber is $hours Hour(s) and $minutes Minute(s) \r\n");
?>